scripts/github_runner/setup_runner.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# MIT License
+#
+# Copyright (c) 2020 Shyam Jesalpura and EASE lab
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+git clone https://github.com/ease-lab/vhive
+cd vhive
+./scripts/travis/setup_node.sh
```

Comment:
you should split the code from setup_node.sh into:
1. Dockerfile stuff (e.g., packages) that should moved to the runner's Dockerfile
2. runtime stuff that must be triggered at the container start. In my understanding, this should be just the devmapper setup.

imagine future maintainers need to dig up into the following sequence: Dockerfile -> setup_node -> setup_system, then compare the packages at the top and at the bottom of the chain

Reply:
done